tests/_support/index.js

```diff
@@ -600,6 +600,28 @@ Cypress.Commands.add("configureCluster", function(configuration) {
       });
   }
 
+  if (configuration.communityPackages) {
+    router
+      // Packages
+      .route({
+        method: "POST",
+        url: /package\/describe/,
+        status: 200,
+        response: "fx:cosmos/community-package-describe"
+      });
+  }
+
+  if (configuration.dependencyPackage) {
```

Comment:
A new configuration I added. It's very similar to the existing one, expect it has a dependency.